src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -671,8 +671,15 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 Return True
             End If
 
-            'If it's not available on an open page, try the common properties
+            'If it's not available on an open page, try the properties
             Dim prop As PropertyDescriptor
+            prop = GetPropertyDescriptor(PropertyName)
```

Comment:
I don't understand this. Why are reading from the Properties now when we didn't have to before? What does this do to old csproj projects?